docs/resources/online_archive.md

```diff
@@ -8,6 +8,8 @@
 
 ~> **IMPORTANT:** There are fields that are immutable after creation, i.e if `date_field` value does not exist in the collection, the online archive state will be pending forever, and this field cannot be updated, that means a destroy is required, known error `ONLINE_ARCHIVE_CANNOT_MODIFY_FIELD`
 
+~> **IMPORTANT:** Support for Online Archive on `GCP` is available in Private Preview. To request access and participate in the Private Preview release of this feature, see the [registration sign-up](https://www.mongodb.com/products/platform/atlas-online-archive#promo).
```

Comment:
Looks good, potentially I would remove `release` from:

> To request access and participate in the Private Preview release of this feature

Reply:
Wording was taken from [here](https://www.mongodb.com/docs/atlas/online-archive/manage-online-archive/). I think its clearer to keep it as is as the alternative wording to me suggests all of private preview will be accessible.